ui/src/api/api.ts

```diff
@@ -83,7 +83,7 @@ export async function getEntities(arxivId: string, getAllEntities?: boolean) {
     type: ENTITY_API_ALL
   } : {};
   const data = await doGet(
-    axios.get(`/api/v0/papers/arxiv:${arxivId}/entities`, { params })
+    axios.get(`/api/v0/papers/arxiv:${encodeURIComponent(arxivId)}/entities`, { params })
```

Comment:
Looks like `axiso` doesn't intend to even support this for you. Boo on them!
https://github.com/axios/axios/issues/706#issuecomment-280777539

Reply:
They'd need a fancier interface than simply accepting the URL as a string, so I can kinda understand their POV. Still annoying that you have to remember to do it though.